<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INFLAP322" failures="2" tests="7" name="managebooking" time="2.453" errors="0" timestamp="2020-12-28T22:09:38 IST">
  <testcase classname="com.restful.booker.tests.ManageBookingTest" name="patchBookingTest" time="0.37">
    <failure type="java.lang.AssertionError" message="the entered booking id does not exist">
      <![CDATA[java.lang.AssertionError: the entered booking id does not exist
at com.restful.booker.tests.ManageBookingTest.patchBookingTest(ManageBookingTest.java:83)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- patchBookingTest -->
  <testcase classname="com.restful.booker.tests.ManageBookingTest" name="patchBookingTest" time="0.388"/>
  <testcase classname="com.restful.booker.tests.ManageBookingTest" name="updateBookingTest" time="0.382">
    <failure type="java.lang.AssertionError" message="the entered booking id does not exist">
      <![CDATA[java.lang.AssertionError: the entered booking id does not exist
at com.restful.booker.tests.ManageBookingTest.updateBookingTest(ManageBookingTest.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- updateBookingTest -->
  <testcase classname="com.restful.booker.tests.ManageBookingTest" name="updateBookingTest" time="0.383"/>
  <testcase classname="com.restful.booker.tests.ManageBookingTest" name="createBookingTest" time="0.428"/>
  <testcase classname="com.restful.booker.tests.ManageBookingTest" name="createBookingTest" time="0.398"/>
  <testcase classname="com.restful.booker.tests.ManageBookingTest" name="deleteBookingTest" time="0.0">
    <skipped/>
  </testcase> <!-- deleteBookingTest -->
</testsuite> <!-- managebooking -->
